hnix.cabal

```
@@ -351,15 +351,15 @@ common shared
     , BangPatterns
     , ViewPatterns
   build-depends:
-      base >= 4.12 && < 4.16
+      base >= 4.12 && < 4.18
     , data-fix >= 0.3.0 && < 0.4
     , exceptions >= 0.10.0 && < 0.11
     , filepath >= 1.4.2 && < 1.5
     , optparse-applicative >= 0.14.3 && < 0.17
-    , relude >= 1.0.0 && < 1.1.0
+    , relude >= 1.0.0 && < 1.1.1
```

Unless there's a special reason for having an upper bound on a minor version, this would be better:

```suggestion
    , relude >= 1.0.0 && < 1.2
```